.github/pull_request_template.md

```diff
@@ -1,23 +1,12 @@
-Fixes # (issue)
+### Fixes #_issue_
 
 ## Description
 
-Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
```

Comment:
One thing I'll make note of is that I removed `List any dependencies that are required for this change.`
I can't really picture when this would be necessary. I'm thinking that it's for complicated situations where the PR owner would be very aware of the need to mention them.